### ISSUE TITLE
Unable to find generator.jar and benchmark.jar

### ISSUE THREAD

**Issue by Vitolearns** [robertoxmed/MC-DAG#2]
Hi，Dr.Medina, I encountered this issue when I  tried  to run bench.py, the function generate() and benchmark() uses jar file 'bin/generator.jar' and 'bin/benchmark.jar', which I am unable to find in your program , could you please upload these two needed jar files?

**Comment by Corleone-p** [comment]
> 嗨，Medina博士，我在尝试运行 bench.py 时遇到了这个问题，函数generate（）和benchmark（）使用jar文件“bin/generator.jar”和“bin/benchmark.jar”，我无法在您的程序中找到，您能上传这两个需要的jar文件吗？

Hello, did you solve the problem?

**Comment by robertoxmed** [comment]
Hi, if you want to use the project you will need to produce these two files from source. A bug was reported recently to me regarding reading and parsing XML files, which should be easily solvable.

**Comment by Corleone-p** [comment]
> 嗨，如果你想使用这个项目，你需要从源代码生成这两个文件。最近有人向我报告了关于读取和解析XML文件的错误，这应该很容易解决。
Thank you for your reply, Dr Medina. The problem has been solved, but I found that your Mixed-Criticality DAG Framework differs from the parameter description in your paper 'Directed Acyclic Graph Scheduling
for Mixed-Criticality Systems' in which the parameters are described differently.

**Comment by robertoxmed** [comment]
It is possible that the description of that paper is outdated, or that the code uses different names. The most recent paper is "Generalized mixed-criticality static scheduling for periodic directed acyclic graphs on multi-core processors"

**Comment by Corleone-p** [comment]
> 该论文的描述可能已经过时，或者代码使用了不同的名称。最近的论文是“Multi-core processors上的周期性有向无环图的广义混合临界静态调度”

Thank you for your patience, Dr Medina.

**Comment by Corleone-p** [comment]
> 嗨，如果你想使用这个项目，你需要从源代码生成这两个文件。最近有人向我报告了关于读取和解析XML文件的错误，这应该很容易解决。

Yes, Dr. Medina, your code does have a problem reading the xml file when calling Main in scheduling.
